packages/office-ui-fabric-react/src/components/Link/Link.types.ts

```diff
@@ -33,6 +33,12 @@ export interface ILinkProps extends React.AllHTMLAttributes<HTMLAnchorElement |
    * Theme (provided through customization.)
    */
   theme?: ITheme;
+
+  /**
+   * Provide an html tag name (one of 'span' | 'a' | 'button' | 'div') to use as the tag
+   * returned from the Link component.
+   */
+  renderAs?: string | React.ComponentClass | React.StatelessComponent;
```

Comment:
why rename this?

`<Link as='div'` seems more simpler than `<Link renderAs='div'`

Reply:
Wouldn't have done it if it wasn't needed, as I like the shorter one better ;) 

But `as` is part of the html attributes. So it's either `string | undefined` which means we can't use custom components like `Route`.

Reply:
Seems like it's part of the html `<link as="video" />` tag, that has no link to our `<Link />` :trollface: 

But since `interface ILinkProps extends React.AllHTMLAttributes<HTMLAnchorElement | HTMLButtonElement | HTMLElement | LinkBase> {` we can't use `as=` unless we change that inheritance.